README.md

```diff
@@ -88,11 +88,11 @@ The database, auth etc logic is here
 
 Built with:
 
-- Nextjs 14
+- [Nextjs 14](https://nextjs.org/docs/app/building-your-application/upgrading/version-14)
 - [Next Auth](https://next-auth.js.org/)
 - [Drizzle ORM](https://drizzle.team/)
 - [Cloudflare D1 database](https://developers.cloudflare.com/d1/get-started/)
-- Cloudflare ratelimiter
+- [Cloudflare ratelimiter](https://developers.cloudflare.com/waf/rate-limiting-rules/)
```

Comment:
the ratelimiting URL is wrong, here's the right one https://developers.cloudflare.com/workers/runtime-apis/bindings/rate-limit/

for nextjs just put nextjs.org

Reply:
Done